src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+spring:
+  application:
+    name: dps-data-compliance
+
+server:
+  port: 8080
+  servlet:
+    context-path: /
+  use-forward-headers: true
```

Comment:
you should have got an intellij warning on this line, replaced with
```
  forward-headers-strategy: native
```

Reply:
that sounds like it would require some fancy paid-for intelliJ version

Reply:
👍 yep, nothing showing for my poor Community edition, but replaced anyway 😄